development/build-image.sh

```diff
@@ -55,7 +55,7 @@ echo "Final image name: $USERNAME/$IMAGE:$TAG"
 
 if [[ $TAG = "develop-$CONFIG_NAME" ]]; then
     git checkout -- docker-compose.yml
```

Comment:
do we need to check out here, Line 65 and Line 67, when we're no longer editing?

Reply:
Does the else condition Line 66-73 need to removed? Seems there's no difference?

Reply:
I think it's worth leaving in just in case someone does add edits inadvertently or if an edit makes it onto our develop branch somehow